.github/workflows/ci.yml

```diff
@@ -45,15 +64,16 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: pnpm
       - run: pnpm install --frozen-lockfile
+      - run: pnpm playwright install
       - run: pnpm test
       - name: archive test results
         if: failure()
         shell: bash
-        run: find packages -type d -name test-results -not -empty | tar -czf test-results.tar.gz --exclude='*-retry*' --files-from=-
+        run: find packages -type d -name test-results -not -empty | tar -czf test-results.tar.gz --exclude='*-retry2*' --exclude='*-retry3*' --exclude='*-retry4*' --exclude='*-retry5*' --files-from=-
```

Comment:
I'd prefer to upload as many logs from flakey tests as possible. I don't think we're in danger of running out of space since old logs get cleared

Reply:
removed the exclusions. still believe this is wasteful but lets see how much space they claim.